app/controllers/api/v1/keybase_proofs_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class Api::V1::KeybaseProofsController < Api::BaseController
+  respond_to :json
+
+  def index
+    @account = Account.find_local!(params[:username])
```

Comment:
gather the avatar here too?

Reply:
haven't done it yet, but it's on my list

Reply:
done

Reply:
done